### ISSUE TITLE
Torch bug

### ISSUE THREAD

**Issue by babybasher** [micdoodle8/Galacticraft#1537]
1. When placing a normal torch on the moon in a sealed area, there is no animated flame.  It's just a glowing ember, gives off the correct light though. I thought it may just be my graphic settings, however this leads to another issue.

2. When breaking the torch to pick it up, they don't stack with the normal torches.  Hovering the cursor over the retrieved torches in my inventory displays 'Torch' in blue text, not the normal white text.

**Comment by radfast** [comment]
1.  Good spot, the flame should be there but it's missing.  We'll take a look at that.

2.  Intended that it's blue.  http://wiki.micdoodle8.com/wiki/Unlit_Torch  What's weird is that __all__ your torches should be the blue kind in Galacticraft dimensions, there shouldn't be any white ones.  Please test what happens if you try to use the white torches, like pick up a stack of those white torches with your mouse and move them to a different inventory slot.  With any luck they'll instantly turn to blue.  But they should be blue from the beginning, looks like this may be a server/client sync issue, again we'll look.

**Comment by babybasher** [comment]
I can't reproduce the 'bug' now :P

Tried making a singleplayer creative world but it all worked fine.

Loaded up my (dedicated) server but I have already thrown the torches away last night :(  When I play again tonight, after the damn twins have gone to bed I'll see if i can reproduce it.

I did take a screen shot before going to bed. This is taken on the overworld, 1 torch from the 'blue' stack, the other from the normal stack. Just to prove im not crazy!

http://postimg.org/image/j7rceldyt/
